source/common/http/http2/codec_impl.cc

```diff
@@ -1272,6 +1272,9 @@ ConnectionImpl::Http2Options::Http2Options(
 
   if (http2_options.allow_metadata()) {
     nghttp2_option_set_user_recv_extension_type(options_, METADATA_FRAME_TYPE);
+    ENVOY_LOG_MISC(trace, "Metadata frame support added to codec.");
```

Comment:
nit prefer ENVOY_CONN_LOG. I think one of the two log messages is OK (I think it makes more sense to say the negative case "Codec does not have Metadata frame support")

Reply:
Changed. Also added this to legacy as well.

Reply:
maybe ENVOY_LOG?

Reply:
Note: I changed this to ENVOY_LOG. All the other log statements in codec were CONNECTION_LOG, however since this class doesn't have a reference to a connection, cannot use CONNECTION_LOG.